### PR TITLE
Add "Contact Us" and "Support" to the user menu

### PR DESCRIPTION
**Pull request by Vinnl**
Note: this builds on #1194, so it's probably easiest to just look at 4405e26236f82ce26c95d427532371c8cc519fa7 and 565cd30173060080178c3115cf802c98bc0006ff (and review that other PR :) ).

~~Depends on https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/25 before merging.~~

### CODE REVIEW

**Review by groovecoder — APPROVED** [review]

File: privaterelay/templates/includes/login.html [hunk 1]
```diff
@@ -49,6 +49,28 @@
 						<img class="" alt="" src="{% static 'images/settings.svg' %}" />
 						{% ftlmsg 'nav-profile-settings' %}
 					</a>
+					{% if user_profile.has_premium %}
+						<a
+							href="{{ settings.FXA_SUPPORT_URL }}?utm_source={{ settings.SITE_ORIGIN }}"
```
Comment (maxxcrawford) [inline privaterelay/templates/includes/login.html:54]: What is the benefit of tracking the `utm_source` here?

Comment (groovecoder) [reply privaterelay/templates/includes/login.html:54]: `rel="noopener noreferrer"` means SUMO won't be able to see in the referrer that this traffic comes from `relay.firefox.com` unless we explicitly send it in the `utm_source` param.

File: privaterelay/templates/settings.html [hunk 2]
```diff
@@ -15,9 +15,23 @@
...
       <li>
         <a
-          href="https://support.mozilla.org/kb/activate-firefox-relay-hide-your-email-address-alias?utm_source={{ settings.SITE_ORIGIN }}"
+          href="{{ settings.FXA_SUPPORT_URL }}?utm_source={{ settings.SITE_ORIGIN }}"
+          target="_blank"
+          rel="noopener noreferrer"
+          title="{% ftlmsg 'settings-meta-contact-tooltip' %}"
+          class="glocal-fxa-settings-link"
+        >
+        <img src="{% static 'images/icon-message-purple.svg' %}" alt="">
```
[the remark this answers is not in the document]
Comment (groovecoder) [reply privaterelay/templates/settings.html]: It's not purple anymore ...
image
https://www.youtube.com/watch?v=5jV-E09efRE

[the remark this answers is not in the document]
Comment (Vinnl) [reply privaterelay/templates/settings.html]: Yes it was supposed to be purple (my [mention of that](https://github.com/mozilla/fx-private-relay/pull/1194#issuecomment-947543529) got lost in the PR shuffle :) ) - I'll remove 928bd39388cc551a08e52402cda36e69d8ddda4a and merge this.

**Pull request merged**